plugins/mcreator-localization/help/default/gui/gui_open_tick.md

```diff
@@ -0,0 +1 @@
+Triggers a procedure every tick while this GUI is opened.
```

Comment:
The proper verb form is: ... while this GUI is open.

Reply:
:heavy_check_mark: Fixed